src/rln_contract.ts

```diff
@@ -102,3 +102,85 @@ export class RLNContract {
     return txRegisterReceipt?.events?.[0];
   }
 }
+
+type CustomQueryOptions = {
+  fromBlock?: number;
+  membersFilter: ethers.EventFilter;
+};
+
+// these value should be tested on other networks
```

Comment:
I tested it only in Goerli.
Do we have deployed RLN contracts on other networks for me to test it, @fryorcraken ? 

For now will try to check on other old contracts since it's not that specific to RLN contract.

Reply:
@rymnc would be the right person to ask. I think not.

Reply:
Please refer to the rln-contracts repo, in the deployments directory all the artifacts are available :)

Reply:
We've deployed to sepolia.

Reply:
https://github.com/vacp2p/rln-contract/blob/main/deployments/allDeployments.json

Reply:
Great, I checked and it works fine